src/main/resources/gui/index.html

```diff
@@ -40,10 +40,12 @@
 <body>
 <script type="text/javascript">
 
+  var report;
```

Comment:
Why storing the report into variable? Isn't better to capture the report from the "result"-element if neccessary?

Reply:
Yeah, that can be done but only if the result-element contains actual HTML and not "innerText" otherwise you will be seeing the HTML as text and not as an actual HTML page. And the variable is global which means it can be accessed by other functions.

Reply:
I want to see it as text because it's XML not html

Reply:
Changed back to innerText. Now the report data gets extracted from the div-box like you required it.